src/errors.rs

```diff
@@ -43,6 +44,7 @@ impl Display for ErrorKind {
             ErrorKind::Message(inner) => Display::fmt(inner, f),
             ErrorKind::Osrmc { code, message }=> write!(f, "Osrmc: {}: {}", code, message),
             ErrorKind::NoRoute => write!(f, "Impossible route between points"),
+            ErrorKind::InvalidCoordinate => write!(f, "Invalid coordinate value"),
             ErrorKind::FfiNul(inner) => Display::fmt(inner, f),
         }
```

Comment:
This tells Rust how to `Display` (i.e. print out) each error that this type exposes.